src/layouts/index.scss

```diff
@@ -16,28 +19,19 @@ body {
 }
 
 section {
-  height: 300px;
+  min-height: 300px;
```

Comment:
Can you extract this to a meaningful variable? Or is it meaningless and a value just randomly chosen? If it's the latter, I'd prefer it wasn't in a variable.

Reply:
It's meaningless, it's just to give some room to stretch the page before the copy is added, just for visual effects.